indexer/packages/kafka/src/batch-kafka-producer.ts

```diff
@@ -28,7 +28,7 @@ export class BatchKafkaProducer {
   constructor(
     topic: KafkaTopics,
     producer: Producer,
-    maxBatchSizeBytes: number = config.KAFKA_MAX_BATCH_WEBSOCKET_MESSAGE_SIZE_BYTES,
```

Comment:
default parameters are resolved when the module is loaded so moved it to pass in the value during instantiation to allow for tests to easily modify the value.